rpcs3/Emu/Cell/lv2/sys_trace.cpp

```diff
@@ -11,60 +11,60 @@ LOG_CHANNEL(sys_trace);
 
 s32 sys_trace_create()
 {
-	sys_trace.todo("sys_trace_create()");
+	fmt::throw_exception("Unimplemented" HERE);
```

Comment:
I don't think that adding exceptions here makes any sense... It's better to add dummy lv2_trace object and return its count.